Intersect.Server/Networking/PacketHandler.cs

```diff
@@ -1590,59 +1593,62 @@ public void HandlePacket(Client client, PickupItemPacket packet)
                 foreach (var itemMap in giveItems)
                 {
                     var tempMap = itemMap.Key;
-                    var toRemove = new List<MapItem>();
-                    foreach (var mapItem in itemMap.Value)
+                    if (tempMap.TryGetInstance(player.MapInstanceId, out var tmpInstance))
```

Comment:
Can this just be inverted and `continue;`d? Rather than indenting everything inside. This will also make this massively easier to review.

Reply:
done